### DISCUSSION TITLE
Latest Mac nightly is corrupt

### DISCUSSION THREAD

**Discussion by ModMike** [supermerill/SuperSlicer#2540] [General]
As the title says, the latest Mac nightly Ius corrupt. When unzipped there is a stop on the app.
![Screen Shot 2022-03-15 at 9 32 18 AM](https://user-images.githubusercontent.com/36581178/158388715-e36db6e9-ad4e-4761-abee-df7b143c1ca4.png)

**Comment by supermerill** [comment]
yes, when it fails it's not set as "failed".
i deleted them.

**Comment by ModMike** [comment]
Thanks. You are doing a great job, will donate.

**Comment by ModMike** [comment]
I am sure I missed it but where can I donate?

**Reply by supermerill** [reply]
there is "sponsor" heart in github , in the top-right list. It contains the links.
